third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1606,6 +2347,303 @@ static void HandleNetworkDidReceiveDataEvent(const base::DictionaryValue& info)
   request_info->second.response_data_received += length;
 }
 
+/** ###########################################################################
+ * DOM
+ * @see https://static.replay.io/protocol/tot/DOM/
+ * @see https://chromedevtools.github.io/devtools-protocol/tot/DOM/
+ * ##########################################################################*/
+
+static LocalFrame* gLocalFrame;
+// NOTE: the InspectorDOMAgent is not as helpful as we would like, since it is
```

Comment:
Nit: Remove commented out code.

Reply:
(As I mentioned on Discord, I saw the comments too late, and kept on going. Providing the fixes on top of my next PR instead, which should go live today.)

Removed [here](https://github.com/replayio/chromium/commit/61b51413bacbab0bd4658b4e59ecb21c40088fa6).